mobile/src/reducers/organization/organization.epics.ts

```diff
@@ -26,7 +50,6 @@ export const loadChiefsEpic$ = (action$: ActionsObservable<LoadDepartmentsFinish
         .map(x => loadEmployeeFinished(x))
```

Comment:
can we probably edit loadChiefsEpic so it maps to series of 'loadEmployee' actions?

Reply:
Fixed